src/xunit.netcore.extensions/Discoverers/ConditionalTestDiscoverer.cs

```diff
@@ -48,16 +48,36 @@ internal class ConditionalTestDiscoverer
                     continue;
                 }
 
-                string[] symbols = conditionMemberName.Split('.');
                 Type declaringType = testMethodDeclaringType;
 
-                if (symbols.Length == 2)
+                // We have qualified type name with the assembly name, something like
+                // [ConditionalFact("System.PlatformDetection, CoreFx.Private.TestUtilities!" + nameof(PlatformDetection.IsNonZeroLowerBoundArraySupported))]
```

Comment:
That you are passing multiple arguments and for ConditionalFact and Theory every argument is a condition to evaluate, so in this case it would look for two different properties: 

* System.PlatformDetection
* CoreFx.Private.TestUtilities!IsNonZeroLowerBoundArraySupported

So that would be two different entries. 

https://github.com/dotnet/buildtools/blob/master/src/xunit.netcore.extensions/Attributes/ConditionalFactAttribute.cs#L16

https://github.com/dotnet/buildtools/blob/master/src/xunit.netcore.extensions/Discoverers/ConditionalTestDiscoverer.cs#L41

So if we want the full name perhaps you need a little more parsing? Maybe the assembly name move it at the beginning, something like: 

`AssemblyName!ClassName.PropertyName`